babel_runner.py

```diff
@@ -0,0 +1,114 @@
+#!/usr/bin/venv python3
+"""Script for handling translations with Babel"""
+from __future__ import annotations
+
+import argparse
+import os
+import subprocess
+
+import tomllib
+
+PROJECT_DIR = os.path.dirname(os.path.abspath(__file__))
+
+# Global variables used by pybabel below
+DOMAIN = "messages"
+COPYRIGHT_HOLDER = "Python Software Foundation"
+LOCALES_DIR = os.path.relpath(os.path.join(PROJECT_DIR, "locales"))
+POT_FILE = os.path.relpath(os.path.join(LOCALES_DIR, f"{DOMAIN}.pot"), PROJECT_DIR)
+SOURCE_DIR = os.path.relpath(
+    os.path.join(PROJECT_DIR, "python_docs_theme"), PROJECT_DIR
+)
+MAPPING_FILE = os.path.relpath(os.path.join(PROJECT_DIR, ".babel.cfg"), PROJECT_DIR)
```

Comment:
It might be nicer to use `pathlib` here? Also, any reason to use relative paths rather than absolute?

Reply:
I was using relative path because the output would be too long because of the absolute path. By using relative paths I achieved `creating catalog locales/pt_BR/LC_MESSAGES/messages.po based on locales/messages.pot` (just 'locales/...' instead of the absolute path).

But using the feedback from both of you, I replaced os with pathlib and used `cwd` keyword argument of `subprocess.run()` in a way that this whole relative path pollution is no longer necessary.